spec/helpers/govuk_personalisation_helper_spec.rb

```diff
@@ -0,0 +1,47 @@
+RSpec.describe GovukPersonalisationHelper do
+  include described_class
+
+  describe "#email_subscription_success_banner_heading" do
+    it "displays the subscription success banner when the 'email-subscription-success' flash is present" do
+      account_flash = request.headers["GOVUK-Account-Session"] = GovukPersonalisation::Flash.encode_session("session-id", %w[email-subscription-success])
+      expect(email_subscription_success_banner_heading(account_flash)).to eq("You’ve subscribed to emails about this page")
```

Comment:
Minor: rather than testing the exact text, you could test that the correct translation has been called. e.g.

expect(email_subscription_success_banner_heading(account_flash)).to eq(I18n.t("email. subscribe_title")